core/src/main/scala/kafka/server/KafkaRaftServer.scala

```diff
@@ -73,7 +73,7 @@ class KafkaRaftServer(
     metrics,
     threadNamePrefix,
     controllerQuorumVotersFuture,
-    new StandardFaultHandlerFactory(),
+    new StandardFaultHandlerFactory()
```

Comment:
This is not necessarily a typo, but rather the "trailing comma" feature, hence I wouldn't add this in this PR.